machines/guides-examples/multi-container-machines.html.markerb

```diff
@@ -151,7 +151,7 @@ There are several ways to deploy multi-container machines on Fly.io. Choose the
 You can define your Machine configuration in a JSON file (e.g., `machine-config.json`) and create the Machine using the Fly CLI:
 
 ```bash
-flyctl machines create -f machine-config.json
+flyctl machines create --machine-config machine-config.json
 ```
 
 You can also launch a multi-container app interactively using `flyctl machine run`:
```

Comment:
@Lucais11 According to our [docs](https://fly.io/docs/flyctl/machine-run/#usage), `fly machine run` takes an `<image>` argument. Like so:
`fly machine run <image> [command] [flags]`


I'm not sure if it's optional, but the fact that's it's omitted could be why it doesn't work.

Could you try verifying if something like this works?
`flyctl machine run <your-image-name> --machine-config cli-config.json`

Reply:
image is now optional, but only if you specify `--machine-config`.

See https://github.com/fly-apps/rate-limiter-demo?tab=readme-ov-file#demo-2a---fly-machine-run-with-precanned-app for a command that you can try that works.